### PR TITLE
Improve api for realtime

### PR DESCRIPTION
**Pull request by julienmathis**
API real time for hosts and services

### CODE REVIEW

**Comment by lpinsivy** [comment]
Part of epics #5596 and #5597

File: www/api/class/centreon_realtime_hosts.class.php [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+                $query .= " ORDER BY h.check_attempt ". $this->order.", h.name ";
+                break;
+            case 'ip':
+                $query .= " ORDER BY IFNULL(inet_aton(h.address), h.address) ". $this->order.", h.name ";
+                break;
+            case 'plugin_output':
+                $query .= " ORDER BY h.output ". $this->order.", h.name ";
+                break;
+            case 'criticality_id':
+                $query .= " ORDER BY isnull ".$this->order.", criticality ".$this->order.", h.name ";
```
Comment (kduret) [inline www/api/class/centreon_realtime_hosts.class.php]: if isnull field is not set, query does not work

File: www/api/class/centreon_realtime_hosts.class.php [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+    }
+
+    /**
+     * Get selected fields by the request
+     *
+     * @return array
+     */
+    protected function getFieldContent()
+    {
+        $tab = split(',', $this->arguments['fields']);
```
Comment (kduret) [inline www/api/class/centreon_realtime_hosts.class.php]: split is deprecated, use explode instead

File: www/api/class/centreon_realtime_hosts.class.php [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+
+    /**
+     * CentreonConfigurationService constructor.
+     */
+    public function __construct()
+    {
+        global $centreon;
+
+        parent::__construct();
+        $this->pearDBMon = new CentreonDB('centstorage');
```
Comment (kduret) [inline www/api/class/centreon_realtime_hosts.class.php]: peardbmon should be initialize in centreon_realtime_base

**Pull request merged**